pkg/webhooks/pytorch/pytorchjob_webhook_test.go

```diff
@@ -284,11 +290,56 @@ func TestValidateV1PyTorchJob(t *testing.T) {
 					specPath.Child("elasticPolicy").Child("nProcPerNode"), specPath.Child("nprocPerNode")),
 			},
 		},
+		"unsupported managedBy controller name": {
```

Comment:
```suggestion
		"attempt to set unsupported managedBy controller name gets rejected": {
```
same for other tests